spec/recipes/default_spec.rb

```diff
@@ -35,15 +35,17 @@
     end.converge(described_recipe)
   end
 
-  # check that the recipres are executed
+  subject { chef_run }
+
+  # check that the recipes are executed
   it 'default should include os-hardening recipes by default' do
-    expect(chef_run).to include_recipe 'os-hardening::packages'
-    expect(chef_run).to include_recipe 'os-hardening::limits'
-    expect(chef_run).to include_recipe 'os-hardening::login_defs'
-    expect(chef_run).to include_recipe 'os-hardening::minimize_access'
-    expect(chef_run).to include_recipe 'os-hardening::pam'
-    expect(chef_run).to include_recipe 'os-hardening::profile'
-    expect(chef_run).to include_recipe 'os-hardening::securetty'
-    expect(chef_run).to include_recipe 'os-hardening::sysctl'
+    is_expected.to include_recipe 'os-hardening::packages'
```

Comment:
@chris-rock as I remember we have everywhere the `expect(chef_run)` syntax. What do you think, is this change Ok for us?

I personally would like to have this changes everywhere (incl. other chef repos) and in the dedicated PR(s).